kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobActionDropdown.js

```diff
@@ -98,16 +98,24 @@ define([
                 return Promise.resolve(false);
             }
 
-            const jobIdList = jobManager.getJobIDsByStatus(
+            const jobList = jobManager.getCurrentJobsByStatus(
```

Comment:
the list of jobs to be cancelled or retried has to include only current jobs, i.e. the most recent retry or the original job, if the job has not been retried.